third_party/move/documentation/book/src/loops.md

```diff
@@ -114,6 +114,45 @@ fun pick(
 }
 ```
 
+## The `for` expression
+
+The `for` expression iterates over a range defined using a `lower_bound` (inclusive) and an `upper_bound` (non-inclusive) of type integer, executing its body for each element. Unlike `while`, `for` is designed for scenarios where the number of iterations is determined by a specific range.
```

Comment:
```suggestion
The `for` expression iterates over a range defined using integer-typed `lower_bound` (inclusive) and `upper_bound` (non-inclusive) expressions, executing its loop body for each element of the range. `for` is designed for scenarios where the number of iterations of a loop is determined by a specific range.
```

Reply:
Thanks. I have made the change in this PR: https://github.com/aptos-labs/developer-docs/pull/46